package.json

```diff
@@ -56,6 +56,7 @@
     "@babel/plugin-transform-typescript": "7.15.8",
     "@babel/preset-env": "7.15.8",
     "@babel/register": "7.15.3",
+    "@types/babel__core": "^7.1.16",
```

Comment:
Please explain this change?
Please do that for other non-straight forward changes, beforehand.

Reply:
![image](https://user-images.githubusercontent.com/8672915/140650839-ed98192a-6519-43b5-98f1-5a925f00e45c.png)

vs

![image](https://user-images.githubusercontent.com/8672915/140650862-dde874fa-0b69-456c-902c-5380c7e7539b.png)

Reply:
Yes, it's a JS file, not TS.
Converting those files to TS is not part of this PR.

Reply:
it doesn't have to be TS for it to have autocompletion, and this change was not making this file typescript, but whatever you want 👍

Reply:
> Converting those files to TS is not part of this PR.

don’t think anything is changed. adding @types shouldn’t be an issue it just helps improve DX in vscode at least.

Reply:
@saihaj Yes, it makes sense, but as separate PR.
It has nothing to do with ESM specifically.

Every commit/PR should be topic-based and revertable.
Also, it doesn't help if `git blame` shows that this package was added for the ESM package.

DX improvements can be done but they should be done in separate PRs with explanations not as random changes on unrelated PRs